server.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+const Promise = require('bluebird');
+const bodyParser = require('body-parser');
+
+const db = require('./db/index.js');
+
+const app = express();
+const port = 3987;
```

Comment:
Don't forget to update the port on the card in Trello.
https://trello.com/c/xz6mSRqC